mrp_repair_expense/README.rst

```diff
@@ -0,0 +1,26 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Mrp Repair Expense
+==================
+
+* This module links the employ expenses with a repair order and also adds a
+shortcut from repairs to his own expenses. 
+
+*It also includes all expense lines on the repair order.
+
+*On the expenses a new field repair ordes is created and the new expenses lines
+had the repair's order analytic account.
```

Comment:
* Expense lines will be linked to a repair order in order to use the orders analytic account for expense lines